cardano-testnet/src/Testnet/Utils.hs

```diff
@@ -17,20 +22,30 @@ import           Cardano.CLI.Shelley.Output
 import           Control.Exception.Safe (MonadCatch)
 import           Control.Monad
 import           Control.Monad.IO.Class
+import           Control.Monad.Trans.Except (runExceptT)
 import           Data.Aeson
+import qualified Data.Aeson as Aeson
+import           Data.Aeson.Key
+import           Data.Aeson.KeyMap
+import qualified Data.ByteString as BS
+import           Data.Text (Text)
 import           Data.Word
 import           GHC.Stack
 import           Options.Applicative
 import           System.Directory (doesFileExist, removeFile)
 
+import           Cardano.Chain.Genesis (GenesisHash (unGenesisHash), readGenesisData)
+
+import qualified Cardano.Crypto.Hash.Blake2b as Crypto
```

Comment:
[Import grouping does not adhere to style guide](https://github.com/input-output-hk/cardano-node-wiki/wiki/Style-guide#imports) 🙂

Reply:
https://github.com/input-output-hk/cardano-node-wiki/pull/6

Reply:
Made changes but opened a PR to update the style doc.